### DISCUSSION TITLE
Cannot read properties of undefined (reading 'forEach')

### DISCUSSION THREAD

**Discussion by selimgokirmak** [massCodeIO/massCode#118] [Q&A] [answered]
i was using v.2.0(probably beta version) and i just updated to v2.8.1. then i tried to load my snippets from db.json but got this error. 
any solution?
![Screenshot 2022-07-10 095412](https://user-images.githubusercontent.com/86617834/178135515-c6c5ee73-c0ac-4aa7-9498-e4776865633c.png)

**Comment by antonreshetov** [comment, marked as answer]
Please see #3
